ekan0ra/queue.py

```diff
@@ -25,21 +25,22 @@ def enqueue(self, nick):
         """Add nick to queue."""
         self.append(nick)
 
-    def dequeue(self, nick):
+    def dequeue(self, nick, all=False):
         """
-        Remove first item matching `nick` in queue.
+        Remove every occurrence of `nick` from queue.
```

Comment:
Your method takes a keyword argument `all`, but your _docstring_ does not explain its effect.

In other words, this method works in two ways:

1. If `all` is `True`, then the dequeue operation will remove all occurrences of `nick` from the queue.
2. If `all` is `False`, then the dequeue operation will only remove the first occurrence of `nick` from the queue.